packages/react-error-overlay/webpack.config.iframe.js

```diff
@@ -39,9 +39,7 @@ module.exports = {
                 babelrc: false,
                 configFile: false,
                 compact: false,
-                presets: [
-                  ['babel-preset-react-app/dependencies', { helpers: true }],
-                ],
+                presets: [['@fs/babel-preset-react-app/dependencies', { helpers: true }]],
```

Comment:
we don't publish this package(react-error-overlay) right now. and we also probably don't need to update the overlay with our emotioned babel anyways. So I'd prefer to remove this particular change.